### DISCUSSION TITLE
docker-compose.yml

### DISCUSSION THREAD

**Discussion by i7phillip** [dani-garcia/vaultwarden#3567] [Q&A]
I'm wondering if there are any other options I can enable within the docker-compose.yml file? Is there a list of variables for the environment? For example, I was able to enable "Groups" by adding ORG_GROUPS_ENABLED: "true". Are there any other options I can enable?

**Comment by BlackDex** [comment]
https://github.com/dani-garcia/vaultwarden/blob/adf67a8ee887d21e104adfa8b6521d7971f5a1f1/.env.template
